### ISSUE TITLE
fluent breaks some compiler messages: Note: {0} uses or overrides a deprecated API

### ISSUE THREAD

**Issue by rogerkeays** [rogerkeays/unchecked#4]
seen in the tomcat and hibernate build
```
Note: {0} uses or overrides a deprecated API
```

**Comment by rogerkeays** [comment]
before:
```
TestAttrErrors.java:6: warning: [removal] deprecated() in Depr has been deprecated and marked for removal
        Depr.deprecated(); // deprecated method
            ^
```
after:
```
warning: [removal] deprecated() in Depr has been deprecated and marked for removal
```

**Comment by rogerkeays** [comment]
replacing the custom log class with a custom diagnosticHandler fixes this problem (commit 6d586e6)
